### DISCUSSION TITLE
New game doesn't start, AdvancedPathfinder.dll failed to load

### DISCUSSION THREAD

**Discussion by nathan0909** [voxeltycoon/issues#947] [Troubleshooting]
**Describe the bug**
A clear and concise description of what the bug is. If applicable, add screenshots or video to help explain your problem.

When I want started a new game, this screenshot appears but the folder it's present when I go to the location. I don't know what change or modify in this folder

![Capture d’écran 2022-01-02 104217](https://user-images.githubusercontent.com/47847279/153745556-5ec34ced-c3e4-41b9-99e8-c29c2dc0870a.png)

**Steps to reproduce**
Steps to reproduce the behavior.

Just start a new game and this appears

**Build version**
Version information can be found at the bottom of the settings window.

Last build

**Platform**
OS name and version.

Window 11

**Reproducibility**
How often an issue reproduces: every time, often, rare, once.

Every time

**Logs/saves**
Attach [logs or saves](https://github.com/voxeltycoon/issues/discussions/650) if applicable.

**Comment by andrewpey** [comment]
This is caused by [this mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2565417798). The mod was not updated to the latest version. Just unsubscribe from it, and the game should work as expected.

You can also try to reach out to the mod's author and ask them to update the mod.
